Sources/MobileCoinClient.swift

```diff
@@ -193,17 +194,21 @@ public final class MobileCoinClient {
         memoType: MemoType = .recoverable,
         amount: Amount,
         fee: UInt64,
+        rng: MobileCoinRng? = nil,
```

Comment:
why not make this parameter non-optional with a default value of `defaultRng` ? Removes the nil coalescing operator stuff below, and gives the same result.